hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -66,6 +73,30 @@ public static HoodieTestTable of(HoodieTableMetaClient metaClient) {
     return new HoodieTestTable(metaClient.getBasePath(), metaClient.getRawFs(), metaClient);
   }
 
+  public static String makeNewCommitTime(int sequence) {
+    return String.format("%09d", sequence);
+  }
+
+  public static String makeNewCommitTime() {
+    return makeNewCommitTime(Instant.now());
+  }
+
+  public static String makeNewCommitTime(Instant time) {
+    return COMMIT_FORMATTER.format(time);
+  }
+
+  public static List<String> makeNewCommitTimes(int num) {
+    return makeNewCommitTimes(num, 1);
+  }
+
+  public static List<String> makeNewCommitTimes(int num, int firstOffsetSeconds) {
+    final Instant now = Instant.now();
+    return IntStream.range(0, num)
+        .mapToObj(i ->
+            makeNewCommitTime(now.plus(i == 0 ? firstOffsetSeconds : firstOffsetSeconds + i, SECONDS)))
+        .collect(Collectors.toList());
+  }
```

Comment:
to replace `HoodieTestUtils#monotonicIncreasingCommitTimestamps(int numTimestamps, int startSecsDelta)`